feat(gen): removed pictures and applied guidelines MTA-7432 (#6818)
scaleway/docs-content: pages/apple-silicon/how-to/create-runner.mdx

    <Lightbox image={image} size="medium" alt="A screenshot of the GitHub.com interface for adding new runners." />

    <Message type="tip">

>       The registration token is **single-use** and **expires 1 hour after generation**. Do not share it or publish it publicly.

Reviewer: ```suggestion
      The registration token is **single-use** and **expires 1 hour after generation**. Do not share or publish it.
```

2. Select the machine you want to update. The runner information overview displays.
3. Open the **Settings** tab.
4. Click **Edit** in the **Runner information** section of the page. A pop-up displays.
    <Message type="tip">

>       GitHub tokens have a limited validity of 1 hour. If your token has expired it will not be possible to register your runner. You have to [retrieve a new token](#retrieving-your-github-token).

Reviewer: ```suggestion
      GitHub tokens have a limited validity of 1 hour. If your token has expired, it will not be possible to register your runner. You have to [retrieve a new token](#retrieving-your-github-token).
```

    </Message>
5. Update your runner credentials and click **Save and reboot**. Your runner configuration is updated, and the runner performs a reboot for the new configuration to take effect.

<Message type="tip">

> Runners rely on dev OS releases and are not kept up to date automatically. Once the runner is delivered, you are responsible for keeping the pre-installed software up to date. To simplify your runner fleet maintenance, we recommend personalizing your runners using CI/CD jobs. This way you can easily dispose of runners running outdated versions and replace them with new ones.

Reviewer: ```suggestion
Runners rely on dev OS releases and are not kept up to date automatically. Once the runner is delivered, you are responsible for keeping the pre-installed software up to date. To simplify your runner fleet maintenance, Scaleway recommends personalizing your runners using CI/CD jobs. This way you can easily dispose of runners running outdated versions and replace them with new ones.
```

</Message>

## Reinstalling a runner

> Reinstalling a server delivered in Dev OS mode is not possible due to the lengthy installation process required for the full Dev OS stack.

Reviewer: ```suggestion
Reinstalling a server delivered in Dev OS mode is not possible due to the long installation process required for the full Dev OS stack.
```
